x/ccv/consumer/keeper/keeper.go

```diff
@@ -341,19 +384,21 @@ func (k Keeper) DeleteOutstandingDowntime(ctx sdk.Context, consAddress string) {
 	store.Delete(types.OutstandingDowntimeKey(consAddr))
 }
 
-// IterateOutstandingDowntime iterates over the validator addresses of outstanding downtime flags
-func (k Keeper) IterateOutstandingDowntime(ctx sdk.Context, cb func(address string) (stop bool)) {
+// IterateOutstandingDowntime iterates over the outstanding downtime flags.
+//
+// Note that the outstanding downtime flags are stored under keys with the following format:
+// OutstandingDowntimeBytePrefix | consAddress
+// Thus, the iteration is in ascending order of consAddresses.
+func (k Keeper) IterateOutstandingDowntime(ctx sdk.Context, cb func(address string)) {
```

Comment:
`IterateAllOutstandingDowntime` since the iterator is always completely exhusted.